template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseMemoryOvercommitThresholdConfigProviderTest.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders.hbase;
+
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.ConfigUtils.config;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.sequenceiq.cloudbreak.template.TemplatePreparationObject;
+import com.sequenceiq.cloudbreak.template.views.HostgroupView;
+import com.sequenceiq.common.api.type.InstanceGroupType;
+
+public class HbaseMemoryOvercommitThresholdConfigProviderTest extends
+    AbstractHbaseConfigProviderTest {
+
+    private final HbaseMemoryOvercommitThresholdConfigProvider underTest = new HbaseMemoryOvercommitThresholdConfigProvider();
+
+    @Test
+    void testMemoryOvercommitThresholdValueWhenDatalake7217() {
```

Comment:
Please add a negative test case as well when the runtime version is something older.